packages/waku/src/lib/plugins/vite-plugin-rsc-resolve-rsdw.ts

```diff
@@ -0,0 +1,14 @@
+import type { Plugin } from 'vite';
+
+export function rscResolveRsdwPlugin(): Plugin {
+  return {
+    name: 'rsc-resolve-rsdw-plugin',
+    enforce: 'pre',
+    async resolveId(id) {
+      if (id === 'react-server-dom-webpack/server.edge') {
+        const resolved = await this.resolve(id);
+        return resolved?.id;
+      }
```

Comment:
This is a workaround for "pnpm monorepo" spec, but I wonder if @hi-ogawa has a nicer solution or #1493 has already fix it.